pages/acmucsp.md

```diff
@@ -0,0 +1,6 @@
+---
+title: ACM-UCSP
+permalink: /acmucsp/
+---
+
+[Ir a página principal de ACM-UCSP](https://ucsp.acm.org/) 
```

Comment:
I don’t think a page is necessary for one link, please delete the page and add the href directly in the navigation bar.

Reply:
Yes, actually I don't like either. I didn't find a way to do it using markdown. Do you know how do it?
Other: We upload to server all files or only html? because if only html I get it.

Reply:
Just change the `navigation.yml` file.
The server only has html files, jekyll transforms the plaintext into html.

Reply:
deleted and when website will be released add on href="https://ucsp.acm.org/"